src/parlant/api/sessions.py

```diff
@@ -210,13 +211,33 @@ class SessionCreationParamsDTO(
     ),
 ]
 
+SessionEventCreationParamsActionField: TypeAlias = Annotated[
+    str,
+    Field(
+        description="A single action from the AI agent, such as 'the the user Thinking..' or 'inform the user that more information is coming'",
```

Comment:
Instead of "the the user is Thinking"
`description='A single action that explains what to say; i.e. "Tell the customer that you're thinking and will be right back with an answer."'`